scripts/runfap.py

```diff
@@ -15,6 +15,13 @@
 class Main(App):
     def init(self):
         self.parser.add_argument("-p", "--port", help="CDC Port", default="auto")
+        self.parser.add_argument(
+            "-l",
```

Comment:
Better change that to a flag meaning "do upload only". Since your approach changes the script's behavior  - "runfap.py" by default no longer runs the FAP itself.

Reply:
Done, I swapped out `-l`/`--launch` with `-n`/`--no-launch`.